.github/workflows/build-test.yaml

```diff
@@ -6,6 +6,8 @@ on:
   pull_request:
     branches: [stable, master]
 
+permissions: {}
```

Comment:
Seems to work :+1: 
I'd add a comment that this disables all permissions - syntax isn't intuitive to me